exercises/practice/bob/tests/bob_test.bal

```diff
@@ -7,218 +7,205 @@ function testStatingSomething() {
     string expected = "Whoa, chill out!";
     test:assertEquals(response, expected);
 }
 
-@test:Config {
-    enable: false
-}
+@test:Config {}
 function testShoutingWithNoExclamationMark() {
     string response = hey("I HATE THE DENTIST");
     string expected = "Whoa, chill out!";
     test:assertEquals(response, expected);
 }
 
-@test:Config {
-    enable: false
-}
+@test:Config {}
 function testStatementContainingQuestionMark() {
     string response = hey("Ending with ? means a question.");
     string expected = "Whatever.";
     test:assertEquals(response, expected);
 }
 
-@test:Config {
-    enable: false
-}
+@test:Config {}
 function testNonLettersWithQuestion() {
     string response = hey(":) ?");
     string expected = "Sure.";
     test:assertEquals(response, expected);
 }
 
-@test:Config {
-    enable: false
-}
+@test:Config {}
 function testPrattlingOn() {
     string response = hey("Wait! Hang on. Are you going to be OK?");
     string expected = "Sure.";
     test:assertEquals(response, expected);
 }
 
-@test:Config {
-    enable: false
-}
+@test:Config {}
 function testSilence() {
     string response = hey("");
     string expected = "Fine. Be that way!";
     test:assertEquals(response, expected);
 }
 
-@test:Config {
-    enable: false
-}
+@test:Config {}
 function testProlongedSilence() {
     string response = hey("          ");
     string expected = "Fine. Be that way!";
     test:assertEquals(response, expected);
 }
 
-@test:Config {
-    enable: false
-}
+@test:Config {}
 function testAlternateSilence() {
     string response = hey("\t\t\t\t\t\t\t\t\t\t");
     string expected = "Fine. Be that way!";
     test:assertEquals(response, expected);
 }
 
-@test:Config {
-    enable: false
-}
+@test:Config {}
 function testMultipleLineQuestion() {
     string response = hey("\nDoes this cryogenic chamber make me look fat?\nNo.");
     string expected = "Whatever.";
     test:assertEquals(response, expected);
 }
 
-@test:Config {
-    enable: false
-}
+@test:Config {}
 function testStartingWithWhitespace() {
     string response = hey("         hmmmmmmm...");
     string expected = "Whatever.";
     test:assertEquals(response, expected);
 }
 
-@test:Config {
-    enable: false
-}
+@test:Config {}
 function testEndingWithWhitespace() {
     string response = hey("Okay if like my  spacebar  quite a bit?   ");
     string expected = "Sure.";
     test:assertEquals(response, expected);
 }
 
-@test:Config {
-    enable: false
-}
+@test:Config {}
 function testOtherWhitespace() {
     string response = hey("\n\r \t");
     string expected = "Fine. Be that way!";
     test:assertEquals(response, expected);
 }
 
-@test:Config {
-    enable: false
-}
+@test:Config {}
 function testNonQuestionEndingWithWhitespace() {
     string response = hey("This is a statement ending with whitespace      ");
     string expected = "Whatever.";
     test:assertEquals(response, expected);
 }
+
+@test:Config {}
```

Comment:
Tests are synced from a shared source: https://github.com/exercism/problem-specifications/blob/main/exercises/bob/canonical-data.json If you feel like new test cases make sense, they could be added, but just wanted to let you know.